packages/cli/src/TagHelpers.ts

```diff
@@ -1,109 +1,128 @@
 import {
-	FindOneOptions,
 	getConnection,
-	In,
 } from "typeorm";
 
 import {
 	Db,
 	ITagDb,
-	IWorkflowDb,
 	ResponseHelper,
 } from ".";
 
-
 // ----------------------------------
-//           validators
+//              utils
 // ----------------------------------
 
 /**
- * Validate whether a tag ID exists so that it can be used for a workflow create or tag update operation.
+ * Type guard for string array.
  */
-export async function validateId(id: string): Promise<void> | never {
-	const findQuery = { where: { id } } as FindOneOptions;
-	const tag = await Db.collections.Tag!.findOne(findQuery);
-
-	if (!tag) {
-		throw new ResponseHelper.ResponseError(`Tag with ID ${id} does not exist.`, undefined, 400);
-	}
+function isStringArray(tags: unknown[]): tags is string[] {
+	return Array.isArray(tags) && !tags.some((value) => typeof value !== 'string');
 }
 
 /**
- * Validate whether a tag name has 1 to 24 characters.
+ * Stringify the ID in every `ITagDb` in an array.
+ * Side effect: Remove `createdAt` and `updatedAt` for a slimmer response.
  */
-export function validateLength(name: string): void | never {
-	if (name.length <= 0 || name.length > 24) {
-		throw new ResponseHelper.ResponseError('Tag name must be 1 to 24 characters long.', undefined, 400);
-	}
+export function stringifyId(tags: ITagDb[]) {
```

Comment:
nitpick - confusing naming.. might be a little clearer
```suggestion
export function getTagsResponse(tags: ITagDb[]) {
```